pontoon/base/views.py

```diff
@@ -436,6 +436,30 @@ def get_translation_history(request):
     return JsonResponse(payload, safe=False)
 
 
+@utils.require_AJAX
+def get_team_comments(request):
+    """Get team comments for given locale."""
+    try:
+        entity = request.GET["entity"]
+        locale = request.GET["locale"]
+    except MultiValueDictKeyError as e:
+        return JsonResponse(
+            {"status": False, "message": "Bad Request: {error}".format(error=e)},
+            status=400,
+        )
+
+    entity = get_object_or_404(Entity, pk=entity)
+    locale = get_object_or_404(Locale, code=locale)
+    comments = Comment.objects.filter(entity=entity, locale=locale)
+
+    payload = []
+
+    for c in comments:
+        payload.append(c.serialize())
```

Comment:
These 4 lines look like a good candidate for [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions):
```python
payload = [c.serialize() for c in comments]
```